app/controllers/greetings_controller.rb

```diff
@@ -1,6 +1,8 @@
+# rubocop:enable
 class GreetingsController < ApplicationController
   def random
     greeting = Greeting.order('RANDOM()').first
     render json: greeting
   end
+  # rubocop:enable
 end
```

Comment:
- [ ] Kindly make sure that the app works properly. You can start by correcting the "App.js" file name. (See attached below)
- [ ] [OPTIONAL] Kindly use the duck pattern in structuring the redux files.
![image](https://user-images.githubusercontent.com/62290545/221585459-820c38aa-2a33-4dda-9cd4-b757bbe955a6.png)

- [ ] I couldn't run the app because some error showed up. Some of the errors occurred because you used npm packages and at the same time didn't include them as dependencies for the project.
![image](https://user-images.githubusercontent.com/62290545/221587118-04bb1eae-e934-4f04-9f40-3ea48df6fbd8.png)